src/test/java/com/github/junit5docker/StartMultipleDockerAnnotationsIT.java

```diff
@@ -0,0 +1,106 @@
+package com.github.junit5docker;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.model.Network;
+import com.github.dockerjava.core.DockerClientBuilder;
+import org.apache.http.HttpStatus;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static com.github.dockerjava.core.DefaultDockerClientConfig.createDefaultConfigBuilder;
+import static com.github.junit5docker.StartMultipleDockerAnnotationsIT.TEST_NETWORK_1;
+import static com.github.junit5docker.StartMultipleDockerAnnotationsIT.TEST_NETWORK_2;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@Dockers({
+    @Docker(image = "faustxvi/simple-two-ports", ports = @Port(exposed = 8081, inner = 8080),
+        networks = {TEST_NETWORK_1, TEST_NETWORK_2}),
+    @Docker(image = "faustxvi/simple-two-ports", ports = @Port(exposed = 8082, inner = 8080),
+        networks = {TEST_NETWORK_1, TEST_NETWORK_2})})
+@DisplayName("Multiple docker containers with multiple networks")
+class StartMultipleDockerAnnotationsIT {
```

Comment:
Thanks a lot for that tests !

We also have the `DefaultDockerClientIT` that you way want to add tests to (for the network). It will allow us to detect misusage of the Docker-java framework more easily (already helped us a lot :) )

Another thing we like to do it to add scenarios to the `index.feature` file. It allows you to document your feature in a more natural way and is used to generate the website (so you won't repeat yourself and the website is always up-to-date).

By the way, it looks to me like you're adding two features : multiple containers and network. I agree that the network doesn't make sens without the multiple container feature. But I think having multiple containers without (shared) networks still makes sense. What do you think ?

Reply:
yes, agreed :)
I'll get working on those